README.md

```diff
@@ -1,7 +1,7 @@
 # [![Build Status](https://travis-ci.org/dart-lang/json_serializable.svg?branch=master)](https://travis-ci.org/dart-lang/json_serializable)
 
-Provides `source_gen` `Generator`s which generate code to
-make it simple to serialize to and from JSON.
+Provides [source_gen] `Generator`s to create code for JSON serialization and
+deserialization.
```

Comment:
Can we clearly state something like "To use this package, you need to install source_gen, by going here." ?

Reply:
There's no install, per se. You just put the right things in your pubspec.

I will make sure to mention that.